doc/src/manual/strings.md

```diff
@@ -268,6 +268,20 @@ In this case, the character `∀` is a three-byte character, so the indices 2 an
 and the next character's index is 4; this next valid index can be computed by [`nextind(s,1)`](@ref),
 and the next index after that by `nextind(s,4)` and so on.
 
+Extraction of a substring using range indexing also expects valid byte indices or error is thrown:
```

Comment:
Maybe "an error" (I'm not a native speaker)?

Reply:
I think you are right.